src/AlevinUtils.cpp

```diff
@@ -560,7 +617,7 @@ namespace alevin {
                         neighbors);
     }
 
-    unsigned int hammingDistance(const std::string s1, const std::string s2){
+    unsigned int hammingDistance(const std::string s1, const std::string s2) {
```

Comment:
Should we be accepting `s1` and `s2` by reference here, rather than value?  So that we have

```
 unsigned int hammingDistance(const std::string& s1, const std::string& s2) {
```